components/dashboard/src/projects/Project.tsx

```diff
@@ -259,6 +262,31 @@ export default function () {
                                     <span>Prebuild</span>
                                 </ItemField>
                             </Item>
+                            {isConsideredInactive && (
+                                <Alert
+                                    type={"warning"}
+                                    onClose={() => {}}
+                                    showIcon={true}
+                                    className="flex rounded mb-2 w-full"
+                                >
+                                    Prebuilds have been paused for this projects because no workspaces were opened for
+                                    the past 7+ days.{" "}
+                                    <a
+                                        href="javascript:void(0)"
+                                        className="gp-link hover:text-gray-600"
+                                        onClick={() => {
+                                            if (project) {
+                                                getGitpodService()
+                                                    .server.triggerPrebuild(project.id, null)
+                                                    .then(() => setIsConsideredInactive(false))
```

Comment:
Two possible UX improvements:

- When the user clicks, disable the link & show a spinner, so that users know that something is actually happening (i.e. clicking worked)?

- Once the action is successful, auto-redirect to the new prebuild that is in progress? (As opposed to just hiding the warning and otherwise nothing seems to have changed)

Reply:
@jankeromnes Some comments regarding the other pages:
- **Project Overview**: I'd keep the alert close to the most relevant page for now, but I agree this could be useful to surface here as this is the project homepage.
-  **List of Prebuilds**: I remember being also confused by this, see  [relevant discussion](https://gitpod.slack.com/archives/C02EN94AEPL/p1651662806162509?thread_ts=1651662123.964389&cid=C02EN94AEPL) (internal), https://github.com/gitpod-io/gitpod/issues/8637, and https://github.com/gitpod-io/gitpod/issues/9111.
- **Specific Cancelled Prebuild**: Makes sense! +1

Reply:
Done.